packages/frontend/src/components/DashboardTiles/DashboardSqlChartTile.tsx

```diff
@@ -100,6 +100,7 @@ const SqlChartTile: FC<Props> = ({ tile, isEditMode, ...rest }) => {
         sql: data?.chart.sql,
         slug: data?.chart.slug,
         limit: data?.chart.limit,
+        additionalQueryKey: [data?.chart.slug, data?.chart.sql, savedSqlUuid],
```

Comment:
This is not related to the PR, and I know this prop already exists. To me, it feels like a hack. Besides, we are already passing `slug` and `sql` to the `viz` method. We might need to consider caching from a different perspective rather than hacking in additional keys every time we use this hook. However, I think we should first wait for this PR to get merged: https://github.com/lightdash/lightdash/pull/11611

Reply:
Yea, this was my bad hack. Sorry. It looks like I'll take it out in the linked PR